src/windows/network_status.c

```diff
@@ -173,14 +187,45 @@ static void window_network_status_invalidate(rct_window *w)
 static void window_network_status_paint(rct_window *w, rct_drawpixelinfo *dpi)
 {
 	window_draw_widgets(w, dpi);
-	gCurrentFontSpriteBase = FONT_SPRITE_BASE_MEDIUM;
-	char buffer[sizeof(window_network_status_text) + 10];
-	char* lineCh = buffer;
-	lineCh = utf8_write_codepoint(lineCh, FORMAT_BLACK);
-	strcpy(lineCh, window_network_status_text);
-	gfx_clip_string(buffer, w->widgets[WIDX_BACKGROUND].right - 50);
+	
 	int x = w->x + (w->width / 2);
 	int y = w->y + (w->height / 2);
-	x -= gfx_get_string_width(buffer) / 2;
-	gfx_draw_string(dpi, buffer, 0, x, y);
+	int width = w->width - (24 + 13);
+	int colour = COLOUR_BLACK;
+
+	//Below is a modified version of gfx_draw_string_centred_wrapped
```

Comment:
Im not on my laptop at the moment so Im not able to compare what I explicitly edited side by side, but I believe it was modified to be able to use a raw string, rather than a string Id.